README.md

```diff
@@ -6,9 +6,23 @@
 [![Nordic ClangFormat](https://img.shields.io/static/v1?label=Nordic&message=ClangFormat&labelColor=00A9CE&color=337ab7)](https://github.com/nrfconnect/sdk-nrf/blob/master/.clang-format)
 [![Zephyr compliance](https://img.shields.io/static/v1?label=Zephry&message=compliance&labelColor=4e109e&color=337ab7)](https://docs.zephyrproject.org/latest/contribute/index.html#coding-style)
 
+It determines NAT timeouts in a cellular networks by sending messages to the [NAT-TestServer](https://github.com/NordicSemiconductor/NAT-TestServer) and waiting for the reply.
+
 This application is built with [sdk-nrf](https://github.com/nrfconnect/sdk-nrf).
 
-Used to test NAT timeouts in different networks, sends messages to [NAT-TestServer](https://github.com/NordicSemiconductor/NAT-TestServer).
+## Getting started
+
+Please help us learn more about NAT timeouts in cellular networks:
+
+1. Find the latest firmware build in the [releases](https://github.com/NordicSemiconductor/NAT-TestFirmware/releases) and flash it onto your nRF9160 Development Kit.
+1. Put in the SIM card of your choice and power on the kit. The test will start automatically. **Do not change the location of the kit during testing, switching mobile cells must be avoided.**
+1. Optionally: you may connect it via USB to observe the test runs in a terminal.
+1. Wait until the test finish (indicated by the 4 LEDs blinking in a rotating pattern). If the TCP test is still running after 24 hours, you can abort it. We generally assume that more than 24 hours means sufficient power savings are provided from the network for the majority of use case scenarios.
```

Comment:
```suggestion
1. Wait until the test finishes (This is indicated by the 4 LEDs,  blinking in a rotating pattern). If the TCP test continues to run for more than 24 hours, you can abort it.  This firmware assumes that a test duration exceeding 24 hours indicates a network providing sufficient power savings for majority of the use case scenarios. 
```

Reply:
> This firmware assumes that a test duration exceeding 24 hours indicates a network providing sufficient power savings for majority of the use case scenarios.

No, that is not a feature of the firmware. It will keep testing forever. There is no upper time limit.

Reply:
```suggestion
1. Wait until the test finishes (This is indicated by the 4 LEDs, blinking in a rotating pattern). If the TCP test continues to run for more than 24 hours, you can abort it. It is generally assumed that a test run duration exceeding 24 hours indicates a network providing sufficient power savings for majority of the use case scenarios.
```